### PR TITLE
ATT-35: Info message telling why when no file upload widget.

### PR DESCRIPTION
**Pull request by reagan-meant**
https://issues.openmrs.org/browse/ATT-35

Added Informative message explaining why upload widget is missing when visit is not started.

### CODE REVIEW

**Review by anshulbh9 — COMMENTED** [review]
@reagan-meant you can add some description about the changes.

**Comment by reagan-meant** [comment]
@anshulbh9 I had thought the title would suffice...but I have added it.

File: omod/src/main/webapp/pages/attachments.gsp [hunk 1]
```diff
@@ -53,14 +53,20 @@ ${ ui.includeFragment("coreapps", "patientHeader", [ patient: patient ]) }
 
 <div id="att-page-main">
 
+ <% if(visit == null){ %>
+
+    <h2>Upload widget is not found because no Visit is started.</h2>
```
Comment (mks-d) [inline omod/src/main/webapp/pages/attachments.gsp]: Could you
1. Use a message to support i18n.
2. Share a screenshot of the end result.

?

**Review by mks-d — CHANGES_REQUESTED** [review]

File: omod/src/main/webapp/pages/attachments.gsp (hunk 1 above)
Comment (reagan-meant) [reply omod/src/main/webapp/pages/attachments.gsp]: Could you point me to a documentation where I can find this? @mks-d  @dkayiwa @ibacher Talk post here https://talk.openmrs.org/t/how-to-use-message-to-support-i18n/27729

Comment (mks-d) [reply omod/src/main/webapp/pages/attachments.gsp]: @reagan-meant an example [here](https://github.com/openmrs/openmrs-module-coreapps/blob/5789accfe48bfd7071b1abad6021ead1da325131/omod/src/main/webapp/fragments/patientdashboard/visitIncludes.gsp#L41) in Core Apps.

Comment (reagan-meant) [reply omod/src/main/webapp/pages/attachments.gsp]: Thanks @mks-d....The following commit was before i realised Ui framework does that in one line....Was trying to do it manually.Lemme send a cleaner commit now

Comment (reagan-meant) [reply omod/src/main/webapp/pages/attachments.gsp]: Screenshot is added to the jira ticket

File: api/src/main/resources/messages.properties [hunk 2]
```diff
@@ -27,4 +27,6 @@ ${project.parent.artifactId}.thumbail.save.error=An error occurred and the attac
 ${project.parent.artifactId}.thumbail.delete.success=The attachment was successfully deleted.
 ${project.parent.artifactId}.thumbail.delete.error=An error occurred and the attachment could not be deleted.
 
-${project.parent.artifactId}.misc.label.enterCaption=Enter a caption
\ No newline at end of file
+${project.parent.artifactId}.misc.label.enterCaption=Enter a caption
+
+${project.parent.artifactId}.noActiveVisit=Upload widget is not found because no Visit is started.
```
Comment (mks-d) [inline api/src/main/resources/messages.properties]: Change this to:
>The file upload is disabled while there is no active visit.

Don't capitalise the word visit.

**Review by mks-d — CHANGES_REQUESTED** [review]

**Review by mks-d — CHANGES_REQUESTED** [review]
Ah also, **do not force push** on your branch ATT-35, just add further commits. A force push would make us loose the history of this PR.

**Review by mks-d — APPROVED** [review]

**Pull request merged**